TESTS/mbed_hal_fpga_ci_test_shield/pwm/main.cpp

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if !DEVICE_PWMOUT
+#error [NOT_SUPPORTED] PWM not supported for this target
+#elif !COMPONENT_FPGA_CI_TEST_SHIELD
+#error [NOT_SUPPORTED] FPGA CI Test Shield is needed to run this test
+#else
+
+#include "utest/utest.h"
+#include "unity/unity.h"
+#include "greentea-client/test_env.h"
+#include "mbed.h"
+
+using namespace utest::v1;
+
+#include "MbedTester.h"
+#include "pinmap.h"
+#include "test_utils.h"
+
+
+#define pwm_debug_printf(...)
```

Comment:
Empty?

Reply:
This is macro for debugging. By default, it does nothing.

Reply:
OK, I was expecting if debug , it does something